VERSION_5.md

```diff
@@ -0,0 +1,183 @@
+# Version 5
+
+## Introduction
+
+GEOSwift was started in May 2015 by Andrea Cremaschi. At this time Swift 1.2 had
+been out for just over a month. Over the years, the libaray and Swift both grew
```

Comment:
Library is spelled wrong.

Reply:
Also I think the comma is superfluous, but I may be wrong on that one.

Reply:
I think the comma is okay: https://english.stackexchange.com/questions/273191/a-question-on-commas